client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-container.tsx

```diff
@@ -18,11 +19,11 @@ const PatternAssemblerContainer = ( {
 }: Props ) => (
 	<BlockRendererProvider siteId={ siteId } stylesheet={ stylesheet }>
 		<PatternsRendererProvider
-			// Use theme demo site to render the site-related blocks for now.
-			// For example, site logo, site title, site tagline, posts.
-			siteId={ siteId }
+			// Use this site to render the posts of query patterns.
+			siteId={ POST_SITE_ID }
```

Comment:
Calling it `PLACEHOLDER_SITE` might be better as it would also affect the site logo (and even the site title and tagline if the `siteInfo` is not provided). What do you think?